### PR TITLE
Add packet mirroring to beta.

### PR DESCRIPTION
**Pull request by ndmckinley**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-resource
`google_compute_packet_mirroring`
```

### CODE REVIEW

File: products/compute/api.yaml [hunk 1]
```diff
@@ -7476,6 +7486,147 @@ objects:
...
+
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: The name of the packet mirroring rule
+        required: true
+      - !ruby/object:Api::Type::String
+        name: description 
+        description: A human-readable description of the rule.
+        input: true
```
Comment (chrisst) [inline products/compute/api.yaml]: This is forceNew?

File: products/compute/api.yaml [hunk 2]
```diff
@@ -7476,6 +7486,147 @@ objects:
...
+            description: |
+                IP CIDR ranges that apply as a filter on the source (ingress) or
+                destination (egress) IP in the IP header. Only IPv4 is supported.
+            item_type: Api::Type::String
+      - !ruby/object:Api::Type::NestedObject
+        name: mirroredResources
+        required: true
+        description: |
+            A means of specifying which resources to mirror.
+        at_least_one_of:
```
Comment (chrisst) [inline products/compute/api.yaml]: I'm not sure the syntax is correct on this: https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3157-old..auto-pr-3157#diff-3ab2ba91e18e29d378d69f103ff962d0R108

File: products/compute/api.yaml (hunk 1 above)
Comment (ndmckinley) [reply products/compute/api.yaml]: It is!  Descriptions go that way most of the time.  It's nuts.

**Comment by ndmckinley** [comment]
My unit test is running into quota issues, so I'm going to submit since running "terraform apply" with this test's config worked during development.  If the test fails in the nightly I'll fix it from there.

**Comment by danawillow** [comment]
TPG `make website-test` is failing due to broken link (probably needs to be beta-guarded), can you fix?

**Comment by ndmckinley** [comment]
Ah, shoot, sure.

**Pull request merged**